lib/ansible/plugins/filter/core.py

```diff
@@ -277,7 +277,12 @@ def get_encrypted_password(password, hashtype='sha512', salt=None, salt_size=Non
 
 
 def to_uuid(string):
-    return str(uuid.uuid5(UUID_NAMESPACE_ANSIBLE, str(string)))
+    return to_uuid5(string, namespace=UUID_NAMESPACE_ANSIBLE)
+
+
+def to_uuid5(string, namespace=UUID_NAMESPACE_ANSIBLE):
+    uuid_namespace = uuid.UUID(namespace)
+    return str(uuid.uuid5(uuid_namespace, str(string)))
```

Comment:
Values returned from filters should use `to_tex()` for Python 2/3 compatibility.
```suggestion
    return to_text(uuid.uuid5(uuid_namespace, str(string)))
```